notifier/settings.py

```diff
@@ -3,6 +3,9 @@
 import os
 import platform
 
+# Change this
+SECRET_KEY = "bI7S2DOOdOLHu3pry1hr2glKlBvBRIB9Uz9CjjI69nOdFyyhsLww06y95z3caIoC0H6RzwLQQe0B9gE43QUM4hjBy6OHhpv1Fwil"
```

Comment:
Does this need to be a real key or could we set this to something like "Please_change_me"?